modules/ztp-image-based-upgrade-generate-seed-image.adoc

```diff
@@ -0,0 +1,267 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The {lcao} will do the following:
+
+// seed cluster, some things are cluster specific that are not included
+// all operator, LCAO, OADP, LVMS, included in the seed image, telco day2 stuff is incl
+// system ocnfig checks ()
+
+* Checks for required system configurations
+* Performs any necessary system cleanup before generating the seed image
+* Launches the image generation task, which involves the following:
+//seedgen Secret and seedgenerator get deleted before imager to not be added to the image
+** Stops the cluster operators
+** Prepares the seed image configuration
+** Generates and pushes the seed image to the image repository specified in the `imageBasedUpgrade` CR
+** Restores the cluster operators
+** Expires seed cluster certifications
+//** Prepares for precaching operations
+//** Backs up platform configurations and artifacts
+* Generates new certificates for the seed cluster
+* Restores and updates the `SeedGenerator` CR on the seed cluster, and the `ManagedCluster` CR on the hub cluster
+
+//TODO Expand on what the seed image contains and explicitly say what it doesn't
+
+.Prerequisites
+
+* You deployed an {sno} with DU profile.
+* You installed the {lcao} on the seed cluster.
+* You installed the OADP Operator on the seed cluster.
+* You are logged in as a user with `cluster-admin` privileges.
+* The seed cluster has the same CPU topology as the target cluster.
+* The seed cluster and the target cluster have the same {rh-rhacm} or multicluster engine version.
+* The seed cluster is registered as a managed cluster.
+// Are there more prereqs?
+
+.Procedure
+
+. Detach the seed cluster.
```

Comment:
Detach the seed cluster either manually or by providing the hubKubeConfig information as shown in the next step. It should be somehow linked with the next step, otherwise, a user could detach the cluster and then add the hubkubeconfig in the next step too.

Reply:
I expanded this to be super clear and now the two ways to detach the cluster is mentioned in two places:

> Detach the seed cluster manually or provide the `hubKubeConfig` information in the `Secret` CR.

The next step is creating the CR and I also highlight in callout 3 that you need to do it manually or populate the field.